docusaurus.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
         },
         {
           label: 'Libraries',
-          to: 'categories/libraries',
+          to: '/#libraries',
```

Comment:
Out of curiosity, what does that change do?

Reply:
It scrolls to `Libraries` on the landing page

Reply:
right :facepalm: